microk8s-resources/wrappers/microk8s-join.wrapper

```diff
@@ -21,7 +16,7 @@ exit_if_no_permissions
 
 if is_strict
 then
-  LD_LIBRARY_PATH=$IN_SNAP_LD_LIBRARY_PATH ${SNAP}/usr/bin/python3 ${SNAP}/scripts/wrappers/join.py "${@}"
+  ${SNAP}/usr/bin/python3 ${SNAP}/scripts/wrappers/join.py "${@}"
 else
-  sudo -E LD_LIBRARY_PATH=$IN_SNAP_LD_LIBRARY_PATH ${SNAP}/usr/bin/python3 ${SNAP}/scripts/wrappers/join.py "${@}"
+  sudo -E ${SNAP}/usr/bin/python3 ${SNAP}/scripts/wrappers/join.py "${@}"
```

Comment:
Don't we drop PATH and LD_LIBRARY_PATH here?

Reply:
thanks, missed it. should be good now